interfaces/apparmor/template.go

```diff
@@ -192,6 +192,11 @@ var defaultTemplate = []byte(`
   /usr/bin/ r,
   /usr/share/distro-info/*.csv r,
 
+  # snapctl and its requirements
+  /usr/bin/snapctl ixr,
+  @{PROC}/sys/net/core/somaxconn r,
+  /run/snapd-public.socket rw,
```

Comment:
@jdstrand took a look at both template changes:

> \<kyrofa\> jdstrand, hey, remember the new snapd socket we discussed yesterday?
> \<kyrofa\> jdstrand, through trial and error, it seems these are the profile changes necessary to make use of it: http://pastebin.ubuntu.com/23085069/ . Can you give me your thoughts when you're able?
> \<jdstrand\> kyrofa: hey, yes
> \<jdstrand\> kyrofa: looks totally fine except please use @{PROC} instead of /proc

I've obviously changed the `/proc` in the paste to `@{PROC}` in this PR.